Formula/ice.rb

```diff
@@ -58,6 +58,7 @@ def install
     args = %W[
       prefix=#{prefix}
       USR_DIR_INSTALL=yes
+      SLICE_DIR_SYMLINK=yes
```

Comment:
Would it be possible to put the `slice` symlink in `#{pkgshare}` (i.e., `#{share}/ice`) instead of `#{share}` since if we ever had a formula named `slice` you'd collide with its `pkgshare`?